src/components/Footer/Footer.scss

```diff
@@ -1,21 +1,48 @@
 @use "../../styles/variables" as *;
 
 .footer {
-  display: flex;
-  align-items: center;
-  justify-content: space-around;
-  color: $color-surface-primary;
-  background-color: $color-brand-primary;
-
-  &__logo-wrapper {
-    padding: 78px 0 0;
+    padding: 80px 0;
     color: $color-surface-primary;
-  }
-
-  &__link-list {
-    display: flex;
-    align-items: center;
-    justify-content: space-around;
-    padding: 18px 0 78px;
-  }
+    background-color: $color-brand-primary;
+
+    &__container {
+        display: flex;
+        padding: 0 68px;
```

Comment:
Move this padding to mixin and apply it here and in pink block of wellbeing section